x-pack/legacy/plugins/alerting_ui/np_ready/public/application/home.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { Route, RouteComponentProps, Switch } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiPageBody, EuiPageContent, EuiSpacer, EuiTab, EuiTabs } from '@elastic/eui';
+import { BASE_PATH, Section, routeToActions } from './constants';
+import { breadcrumbService } from './lib/breadcrumb';
+import { docTitleService } from './lib/doc_title';
+import { useAppDependencies } from './index';
+
+import { ActionsList } from './sections/actions_list/components/actions_list';
+
+interface MatchParams {
+  section: Section;
+}
+
+export const AlertsUIHome: React.FunctionComponent<RouteComponentProps<MatchParams>> = ({
+  match: {
+    params: { section },
+  },
+  history,
+}) => {
+  const {
+    plugins: { capabilities },
+  } = useAppDependencies();
+
+  const canShowActions = capabilities.get().actions.show;
+  const tabs: Array<{
+    id: Section;
+    name: React.ReactNode;
+  }> = [];
+
+  if (canShowActions) {
+    tabs.push({
+      id: 'actions',
+      name: (
+        <FormattedMessage
+          id="xpack.alertingUI.home.repositoriesTabTitle"
+          defaultMessage="Actions"
+        />
+      ),
+    });
+  }
+
+  const onSectionChange = (newSection: Section) => {
+    history.push(`${BASE_PATH}/${newSection}`);
+  };
+
+  // Set breadcrumb and page title
+  useEffect(() => {
+    breadcrumbService.setBreadcrumbs(section || 'home');
+    docTitleService.setTitle(section || 'home');
+  }, [section]);
+
+  return (
+    <EuiPageBody>
+      <EuiPageContent>
+        <EuiTabs>
+          {tabs.map(tab => (
+            <EuiTab
+              onClick={() => onSectionChange(tab.id)}
+              isSelected={tab.id === section}
+              key={tab.id}
+              data-test-subj="tab"
+            >
+              {tab.name}
+            </EuiTab>
+          ))}
+        </EuiTabs>
+
+        <EuiSpacer size="s" />
+
+        <Switch>
+          {canShowActions && <Route exact path={routeToActions} component={ActionsList} />}
```

Comment:
I'm not sure about this but maybe `canShowActions` should be wrapping the `<Switch>`?

Reply:
There will be more routes added later on and only the one above will be related to `canShowActions`